packages/hash/api/src/model/ontology/data-type.model.ts

```diff
@@ -162,24 +149,29 @@ export default class {
    *   Depends on the RFC captured by:
    *   https://app.asana.com/0/1200211978612931/1202464168422955/f
    *
-   * @param params.accountId the accountId of the account making the update
    * @param params.schema a `DataType`
    */
   async update(
     graphApi: GraphApi,
     params: {
-      accountId: string;
       // we have to manually specify this type because of 'intended' limitations of `Omit` with extended Record types:
       //  https://github.com/microsoft/TypeScript/issues/50638
       //  this is needed for as long as DataType extends Record
       schema: Pick<DataType, "kind" | "title" | "description" | "type"> &
         Record<string, any>;
     },
   ): Promise<DataTypeModel> {
-    const { accountId, schema } = params;
+    const { schema } = params;
 
     const updateArguments: UpdateDataTypeRequest = {
-      accountId,
+      /**
+       * @todo: let caller update who owns the type, or create new method dedicated to changing the owner of the type
+       * @see https://app.asana.com/0/1202805690238892/1203063463721791/f
```

Comment:
This linked task doesn't seem to be the intended task (it's the same as the todo below).

Reply:
Thanks for spotting, I intended on linking [this](https://app.asana.com/0/1202805690238892/1203063463721793/f). Must've made a mistake with my clipboard, have addressed in https://github.com/hashintel/hash/pull/1129/commits/5b6dd0e31c8095aab8984efaaa830dd733e1d081